.pre-commit-config.yaml

```diff
@@ -14,3 +14,6 @@ repos:
     rev: "v2.7.1"
     hooks:
       - id: prettier
+        additional_dependencies:
+          - prettier@2.7.1
+          - prettier-plugin-organize-imports@3.1.1
```

Comment:
Technically unsafe for a formatter to do this since imports can have side effects. Not that we meant to rely on those anyhow

Reply:
Interesting. I shouldn't be relying on order, so should be okay.